python/foglamp/services/core/api/plugin_discovery.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from aiohttp import web
+from foglamp.common.plugin_discovery import PluginDiscoveryInstalled
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_help = """
+    -------------------------------------------------------------------------------
+    | GET             | /foglamp/plugins/installed                                |
```

Comment:
don't we want not installed but available too?

Reply:
As @MarkRiddoch mentioned, it will taken up in a future sprint.